test/plugins/commands.test.js

```diff
@@ -1091,6 +1095,70 @@ describe('command plugin test', () => {
             });
         });
 
+        it('creates commands tag if has good permission and tag does not exist with tag', () => {
```

Comment:
command tags?

Reply:
fixed to `a command tag` because this test creates just one tag.